scene/gui/control.cpp

```diff
@@ -480,10 +480,10 @@ void Control::_notification(int p_notification) {
 			if (is_set_as_toplevel()) {
 				data.SI=get_viewport()->_gui_add_subwindow_control(this);
 
-				/*if (data.theme.is_null() && data.parent && data.parent->data.theme_owner) {
+				if (data.theme.is_null() && data.parent && data.parent->data.theme_owner) {
 					data.theme_owner=data.parent->data.theme_owner;
 					notification(NOTIFICATION_THEME_CHANGED);
-				}*/
+				}
```

Comment:
Shouldn't you change it to use `_propagate_theme_changed` as all the rest of the code does?